proto/storage/cluster.proto

```diff
@@ -250,6 +250,22 @@ message UpgradeProgress {
     google.protobuf.Timestamp since    = 3;
 }
 
+message ClusterCVEEdge {
```

Comment:
Can we please move it back to `relations.proto`

Reply:
It's done. Along with the move of the proto message in the proto.lock file (which caused `make storage-protos-compatible` or `make style` to fail locally).